src/components/common/CardRow/CardRow.stories.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Story, Meta } from '@storybook/react/types-6-0'
+import styled from 'styled-components'
+
+import { GlobalStyles, ThemeToggler } from 'storybook/decorators'
+
+import { Card } from '../Card/index'
+import { CardRow, CardRowProps } from '.'
+
+export default {
+  title: 'Common/CardRow',
+  component: CardRow,
+  decorators: [GlobalStyles, ThemeToggler],
+} as Meta
+
+const CardBody = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  p {
+    margin-bottom: 0px;
```

Comment:
In order to center the content we can set this margin in 0: (`margin: 0` instead of `margin-botton:0px`)
![image](https://user-images.githubusercontent.com/622217/129785617-9b4ff989-815f-432c-8467-f38d1fe61596.png)

Then we need to add 
    `display: flex;
    justify-content: center;
    align-items: center;`
![image](https://user-images.githubusercontent.com/622217/129786089-6727d65a-af58-447a-8289-ae5d0597fecd.png)
****